src/odemis/driver/test/xt_client_test.py

```diff
@@ -724,108 +742,90 @@ def test_apply_autostigmator(self):
-            time.sleep(0.01)
-            self.assertEqual(auto_contrast_brightness_state, True)
-            self.assertIsInstance(auto_contrast_brightness_future, ProgressiveFuture)
-
-            # Stop auto auto contrast brightness and check if it stopped running.
-            auto_contrast_brightness_future.cancel()
-            time.sleep(5.0)  # Give microscope/simulator the time to update the state
-            auto_contrast_brightness_state = self.microscope.is_running_auto_contrast_brightness(channel)
-            self.assertEqual(auto_contrast_brightness_state, False)
-            self.assertIsInstance(auto_contrast_brightness_future, ProgressiveFuture)
-
-            # Test starting auto contrast brightness and cancelling directly
-            auto_contrast_brightness_future = self.scanner.applyAutoContrastBrightness(role)
-            auto_contrast_brightness_future.cancel()
-            time.sleep(5.0)  # Give microscope/simulator the time to update the state
-            auto_contrast_brightness_state = self.microscope.is_running_auto_contrast_brightness(channel)
-            self.assertEqual(auto_contrast_brightness_state, False)
-            self.assertIsInstance(auto_contrast_brightness_future, ProgressiveFuture)
-
-            # Start auto contrast brightness
-            max_execution_time = 60  # Approximately 3 times the normal expected execution time (s)
-            starting_time = time.time()
-            auto_contrast_brightness_future = self.scanner.applyAutoContrastBrightness(role)
-            time.sleep(0.5)  # Give microscope/simulator the time to update the state
-            # Wait until the auto contrast brightness is finished
-            auto_contrast_brightness_future.result(timeout=max_execution_time)
-
-            # Check if the time to perform the auto contrast brightness is not too long
-            self.assertLess(time.time() - starting_time, max_execution_time,
-                            "Execution of auto contrast brightness was stopped because it took more than %s seconds."
-                            % max_execution_time)
-
-            auto_contrast_brightness_state = self.microscope.is_running_auto_contrast_brightness(channel)
-            self.assertEqual(auto_contrast_brightness_state, False)
+        # Start auto contrast brightness and check if it is running.
+        auto_contrast_brightness_future = self.scanner.applyAutoContrastBrightness(self.detector.role)
+        auto_contrast_brightness_state = self.microscope.is_running_auto_contrast_brightness(self.scanner.channel)
+        time.sleep(0.01)
+        self.assertEqual(auto_contrast_brightness_state, True)
+        self.assertIsInstance(auto_contrast_brightness_future, ProgressiveFuture)
+
+        # Stop auto auto contrast brightness and check if it stopped running.
+        auto_contrast_brightness_future.cancel()
+        time.sleep(5.0)  # Give microscope/simulator the time to update the state
```

Comment:
These changes are just changes in the indent because of the removal of the for loop and the DETECTOR2CHANNELNAME.
I suppose that in the previous merge (last year) I did check that 5 seconds is needed.